features/step_definitions/ssp_steps.rb

```diff
@@ -22,15 +22,23 @@
 end
 
 Given /^I am at the g7 services page$/ do
-  visit "#{dm_frontend_domain}/suppliers/frameworks/g-cloud-7/services"
+  #visit "#{dm_frontend_domain}/suppliers/frameworks/g-cloud-7/services"
```

Comment:
These commented-out lines can probably be deleted if the generalised version is working OK?